--- a/third_party/lit/v3_0/BUILD.gn
+++ b/third_party/lit/v3_0/BUILD.gn
@@ -28,6 +28,7 @@
     "//chrome/browser/resources/access_code_cast:build_ts",
     "//chrome/browser/resources/accessibility_annotator_info:build_ts",
     "//chrome/browser/resources/accessibility_annotator_internals:build_ts",
+    "//roamux/app/resources/settings_about:build_ts",
     "//chrome/browser/resources/actor_overlay:build_ts",
     "//chrome/browser/resources/ai_overlay_dialog:build_ts",
     "//chrome/browser/resources/app_home:build_ts",
